src/app/pages/survey-fill/survey-fill.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="w-full bg-primary">
+  <div class="content pt-16 pb-16 text-white">
+    <app-breadcrumbs [segments]="breadcrumbs"></app-breadcrumbs>
```

Comment:
IMO we can drop the breadcrumbs as we don't have/want a survey index page, so it's not useful. Probably safe to drop the horizontal line as well

![image](https://github.com/user-attachments/assets/b504393a-cc05-48b7-afec-5b69ba2da214)